### PR TITLE
updated signinapp tests for coverage

### PR DESCRIPTION
**Pull request by acaruso-oddball**
## Summary

Fixes code coverage for the SignInApp component so that all statements, branches, functions, lines are a minimum 90%.

## Related issue(s)
[Fix SignInApp code coverage](https://github.com/department-of-veterans-affairs/va.gov-team/issues/87767)

## Testing done

Run yarn test:unit <location-of-unit-test> --coverage --coverage-html
Open up the coverage/index.html file in a browser
Verify code coverage minimums and refactor as needed

## What areas of the site does it impact?

Testing around authentication components. 

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed

### CODE REVIEW

**Review by micahchiang — CHANGES_REQUESTED** [review]
From what I can tell, the package added to package.json and yarn.lock in this PR isn't necessary. Could you please revert? Thanks!

**Comment by acaruso-oddball** [comment]
Hi folks, I've reverted the changes as I encountered the following warning during testing:

` Browserslist: caniuse-lite is outdated. Please run:
npx browserslist@latest --update-db
Why you should do it regularly: https://github.com/browserslist/browserslist#browsers-data-updating `

Although updating as suggested was a potential solution, I confirmed that it was not necessary to resolve the issue.

File: package.json
[the remark this answers is not in the document]
Comment (acaruso-oddball) [reply package.json:1]: removed thx

File: yarn.lock
[the remark this answers is not in the document]
Comment (acaruso-oddball) [reply yarn.lock:1]: removed thx

**Review by acald-creator — APPROVED** [review]

**Review by micahchiang — APPROVED** [review]

**Pull request merged**